tools/IoTEdgeDevOps/DevOpsLib/BugQuery.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+using System;
+
+namespace DevOpsLib
+{
+    public class BugQuery
+    {
+        public BugQuery(string title, string area, BugPriority priority, bool inProgress)
+        {
+            ValidationUtil.ThrowIfNullOrWhiteSpace(title, nameof(title));
+            ValidationUtil.ThrowIfNullOrWhiteSpace(area, nameof(area));
+            ValidationUtil.ThrowIfNull(inProgress, nameof(inProgress));
+
+            this.Title = title;
+            this.Area = area;
+            this.Priority = priority;
+            this.InProgress = inProgress;
+        }
+        public string Title { get; }
```

Comment:
I think the title should be derived inside this class, not passing in.

Reply:
I used computed properties to achieve this